src/code.cpp

```diff
@@ -452,6 +452,7 @@ Expr *read_code() {
 	              pref = new string("mp_");
 	              break;
 	            }
+                    break;
```

Comment:
Just a note about the weird indentation: the part of the file currently contains a mix of tabs and spaces, so it's hard to give this statement an indentation level that looks good in every editor/viewer.

I'll run `clang-format` on the whole repository sometime soon.

Reply:
Yeah, this should probably be cleaned up, but this seems fine for this PR.

Reply:
Running `clang-format` in a separate PR may be a good idea, it is really a big mess right now...